gdal/swig/include/java/typemaps_java.i

```diff
@@ -62,6 +62,10 @@
 
 %typemap(javain) SWIGTYPE *DISOWN "$javaclassname.getCPtrAndDisown($javainput)"
 
+%typemap(in, numinputs=0) JNIEnv *jenv %{
```

Comment:
that typemap and the one later should probably be removed for other ones, as mentioned above